spec/lucky/force_ssl_handler_spec.cr

```diff
@@ -54,26 +54,28 @@ describe Lucky::ForceSSLHandler do
       context.response.headers["Location"]?.should be_nil
     end
 
-    it "enables HSTS with 1 year max age and subdomains by default" do
+    it "allows setting Strict-Transport-Security" do
```

Comment:
I think using the term `Strict-Transport-Security` makes this easier to understand/google. `hsts` looks odd when configuring the handler